src/api.js

```diff
@@ -54,19 +54,9 @@ class DockAPI {
 
     this.address = address || this.address;
 
-    // Polkadot-js needs these extra type information to work. Removing them will lead to
-    // an error. These were taken from substrate node frontend template.
-    const extraTypes = {
```

Comment:
Removed them as they are already present in types.json

Reply:
good spot